src/Plugin/Field/FieldType/ColorSchemeItem.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\oe_color_scheme\Plugin\Field\FieldType;
+
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\Core\TypedData\DataDefinition;
+
+/**
+ * Plugin implementation of the 'Color Scheme' field type.
+ *
+ * @FieldType(
+ *   id = "oe_color_scheme",
+ *   label = @Translation("Color Scheme"),
+ *   module = "oe_color_scheme",
+ *   description = @Translation("Stores a colour scheme name."),
+ *   default_widget = "oe_color_scheme_widget"
+ * )
+ */
+class ColorSchemeItem extends FieldItemBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function schema(FieldStorageDefinitionInterface $field_definition) {
+    return [
+      'columns' => [
+        'name' => [
+          'type' => 'varchar',
+          'length' => 255,
+        ],
+      ],
+      'indexes' => [
+        'format' => ['name'],
```

Comment:
its customary to add it but there should not be so many values that it would make sense